.github/workflows/label-issue-open.yml

```diff
@@ -30,3 +41,27 @@ jobs:
         if: ${{ steps.latest-issue.outputs.hasLabels == 'false'}}
         with:
           labels: 'status: awaiting triage'
+
+  community:
+    name: Add community contribution label
+    runs-on: ubuntu-20.04
```

Comment:
```suggestion
    runs-on: ubuntu-20.04
    needs: 'triage'
```

Reply:
Since all jobs in a workflow are run concurrently we need to declare a dependency here in order to have this job run after the one with the wait.